doc/reference.bib

```diff
@@ -0,0 +1,37 @@
+@Manual{R,
```

Comment:
We should also reference all of the python packages that we used, python itself, as well as our dataset.

Reply:
Also it looks like these references aren't actually used in the .Rmd file.